pybb/migrations/0002_auto_20150424_0918.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pybb', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='category',
+            name='slug',
+            field=models.SlugField(max_length=100, verbose_name='Piece of URL', blank=True),
+            preserve_default=True,
```

Comment:
This arg has been added with django 1.7.1. Its default is `True` ([cf django 1.7.x code source](http://django.readthedocs.org/en/1.7.x/_modules/django/db/migrations/operations/fields.html)
As you use the default value, you should remove this arg to avoid Exception for people who use django 1.7.0.